src/client/src/pages/Admin/Components/AnalysisBox.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import {
+    Button,
+    Grid,
+    Paper,
+    Typography,
+    CardContent,
+} from "@material-ui/core";
+import _ from "lodash";
+import AnalysisTable from "./AnalysisTable";
+
+export default function AnalysisBox(props) {
+    const { handleExecute, lastExecution, statistics } = props;
+
+    return (
+        <Grid container item direction="column" spacing={3} sm={6}>
+            <Grid item>
+                <Paper>
+                    <CardContent>
+                        <Typography variant="h5" style={{ paddingBottom: 5 }}>
+                            Run New Analysis
+                        </Typography>
+                        <form onSubmit={handleExecute}>
+                            <Button
+                                type="submit"
+                                variant="contained"
+                                color="primary"
+                            >
+                                Run Data Analysis
+                            </Button>
+                        </form>
+                    </CardContent>
+                    {!_.isEmpty(statistics) && (
+                        <AnalysisTable
+                            lastExecution={lastExecution}
+                            tableData={statistics}
+                        />
```

Comment:
I'm not sure we needed this table as its own component. I did it before I separated the parent components from `Admin.js`. I think it's fine, but it might be preferable to remove this as its own component.